README.md

```diff
@@ -159,21 +159,26 @@ An array that contains the types of the methods. Default: `['DELETE', 'GET', 'HE
 ## websocket
 
 This module has _partial_ support for forwarding websockets by passing a
-`websocket` option. All those options are going to be forwarded to
-[`@fastify/websocket`](https://github.com/fastify/fastify-websocket).
+`websocket` option. 
```

Comment:
Can you clarify what the `websocket` option is? An object? An array? It is stated for the other options in the documentation.

Reply:
I've improved the docs.